modules/ecr/locals.tf

```
@@ -0,0 +1,59 @@
+locals {
+  primary_prefixes   = ["develop", "main"]
+  secondary_prefixes = ["bug", "chore", "feature"]
+
+  all_prefixes = concat(local.primary_prefixes, local.secondary_prefixes)
+
+  //Sets max amount of the latest develop images to be kept
```

We are going to keep not only develop images
```suggestion
  // Sets max amount of the latest images to be kept
```